src/plugin.test.ts

```diff
@@ -0,0 +1,5 @@
+import plugin from './plugin'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/richardvanbergen/payload-plugin-cloud-storage/pullRequest?prid=7829468)
```suggestion
import plugin from "./plugin"
```